src/renderer/features/governance/components/ReferendumDetails/Timeline.tsx

```diff
@@ -0,0 +1,62 @@
+import { useStoreMap, useUnit } from 'effector-react';
+import { format } from 'date-fns';
+
+import { useI18n } from '@app/providers';
+import { ReferendumId } from '@shared/core';
+import { FootnoteText, OperationStatus, Shimmering } from '@shared/ui';
+import { ReferendumTimelineRecordStatus } from '@shared/api/governance';
+import { detailsAggregate } from '../../aggregates/details';
+
+type Props = {
+  referendumId: ReferendumId;
+};
+
+const getStatusPalette = (status: ReferendumTimelineRecordStatus) => {
+  switch (status) {
+    case 'Cancelled':
+    case 'Rejected':
+    case 'Killed':
+      return 'error' as const;
+
+    case 'Confirmed':
+    case 'Executed':
+    case 'Awarded':
+      return 'success' as const;
+
+    default:
+      return 'default' as const;
+  }
+};
+
+export const Timeline = ({ referendumId }: Props) => {
+  const { t } = useI18n();
+
+  const loading = useUnit(detailsAggregate.$isTimelinesLoading);
```

Comment:
I think we should discuss naming for boolean, mostly we use `is` prefix

Reply:
Changed name to `isLoading` for now